core-bundle/src/EventListener/GlobalsMapListener.php

```diff
@@ -31,7 +31,7 @@ public function onInitializeSystem(): void
     {
         foreach ($this->globals as $key => $value) {
             if (\is_array($value) && isset($GLOBALS[$key]) && \is_array($GLOBALS[$key])) {
-                $GLOBALS[$key] = array_replace_recursive($GLOBALS[$key], $value);
+                $GLOBALS[$key] = array_replace_recursive($value, $GLOBALS[$key]);
```

Comment:
Maybe rename the listener so that it is clear it is only meant for `TL_CTE` and `TL_MOD`?

Reply:
This is to give content elements registered via the `config.php` precedence. The idea behind this is, that people can opt out of the new system by recreating the entry in their `config.php` file. Also, if someone or an extension registered an element instead of the core's element, this would still work.

Reply:
> Maybe rename the listener so that it is clear it is only meant for TL_CTE and TL_MOD?

:+1: Something like `MapContentElementAndFrontendModuleGlobalDefinitionsListener`? :stuck_out_tongue:

Reply:
Ah, well I don't mind the name but a comment would be great 😅

Reply:
Agreed, I added a comment in 9cdfa470c1b141986645e5bd4457e266f912b6d1